hypertrace-graphql-labels-schema/src/main/java/org/hypertrace/graphql/label/request/LabelsConfigRequestBuilderImpl.java

```diff
@@ -0,0 +1,73 @@
+package org.hypertrace.graphql.label.request;
+
+import java.util.Map;
+import javax.inject.Inject;
+import lombok.Value;
+import lombok.experimental.Accessors;
+import org.hypertrace.core.graphql.common.utils.attributes.AttributeScopeStringTranslator;
+import org.hypertrace.core.graphql.context.GraphQlRequestContext;
+import org.hypertrace.core.graphql.deserialization.ArgumentDeserializer;
+import org.hypertrace.graphql.label.deserialization.LabelIdArgument;
+import org.hypertrace.graphql.label.deserialization.LabelKeyArgument;
+import org.hypertrace.graphql.label.schema.Label;
+
+public class LabelsConfigRequestBuilderImpl implements LabelsConfigRequestBuilder {
+  private final ArgumentDeserializer argumentDeserializer;
+
+  @Inject
+  LabelsConfigRequestBuilderImpl(
+      ArgumentDeserializer argumentDeserializer,
+      AttributeScopeStringTranslator scopeStringTranslator) {
+    this.argumentDeserializer = argumentDeserializer;
+  }
+
+  @Override
+  public LabelCreateRequest buildCreateRequest(
+      GraphQlRequestContext requestContext, Map<String, Object> arguments) {
+    return new LabelCreateRequestImpl(
+        requestContext,
+        this.argumentDeserializer
+            .deserializePrimitive(arguments, LabelKeyArgument.class)
+            .orElseThrow());
+  }
+
+  @Value
+  @Accessors(fluent = true)
```

Comment:
Can we shift all the private classes below, and move the create/update/delete functions together?